### DISCUSSION TITLE
Added: LDM notes for June 10, 2020

### DISCUSSION THREAD

**Discussion by orthoxerox** [dotnet/csharplang#3558] [General]
https://github.com/dotnet/csharplang/blob/0f56445e250ddf82b88848b94c59870f13ab8ffc/meetings/2020/LDM-2020-06-10.md

The topic was roles and extensions, #1711

**Comment by orthoxerox** [comment]
According to @CyrusNajmabadi, the LDM started with 9 core scenarios that they've heard lots of feedback on. I would love to see them documented and enumerated publicly, if possible.

**Comment by agocke** [comment]
I'll ping @MadsTorgersen for his examples, since I didn't take notes on the slides 😄
